ios/SimplSdk/SimplSdk.m

```diff
@@ -10,7 +10,28 @@ @implementation SimplSdk
 
 RCT_EXPORT_MODULE()
 
-RCT_EXPORT_METHOD(isApproved:(NSString *)merchantId mobileNumber:(NSString *)mobileNumber emailId:(NSString *)emailId isSandbox:(BOOL)isSandbox successCallback:(RCTResponseSenderBlock)successCallback errorCallback:(RCTResponseSenderBlock)errorCallback) {
+RCT_EXPORT_METHOD(isUserApproved:(NSDictionary *)approvalParams successCallback:(RCTResponseSenderBlock)successCallback errorCallback:(RCTResponseSenderBlock)errorCallback){
+    @try{
+        [GSManager initializeWithMerchantID: approvalParams[@"merchantId"]];
+        [GSManager enableSandBoxEnvironment: approvalParams[@"isSandbox"]];
+        self.user= [[GSUser alloc] initWithPhoneNumber:approvalParams[@"mobileNumber"] email:approvalParams[@"emailId"]];
```

Comment:
why is this `phone_number` and `email` in android and `mobileNumber` and `emailId` in iOS? How will the same code work across platforms?